ush/ufsda/archive.py

```diff
@@ -23,11 +23,11 @@ def atm_diags(config):
     obs_list_yaml = config['OBS_LIST']
     obs_list_config = Configuration(obs_list_yaml)
     obs_list_config = ufsda.yamltools.iter_config(config, obs_list_config)
-    for ob in obs_list_config['observations']:
+    for ob in obs_list_config['observers']:
```

Comment:
I need to think about this more, but I feel like this whole function is not correct. It is archiving diags, but the `type` is `ob`, which I think it should be `diag`. @RussTreadon-NOAA do you have an R2D2 DB where the output from your cycled experiment is stored to look at its structure?

Reply:
Sample output is in `/work2/noaa/da/rtreadon/R2D2`.   For example, output for 2021-12-30 is in 
- VAR:  `ncdiag/ob/prgdasens/PT6H/2021-12-30/`
- ENS:  `ncdiag_lgetkf/ob/prgdasens/PT6H/2021-12-30/`

Reply:
ok yeah so this isn't "correct" then, I think it should be storing them in a path such as:
`gfs/diag/prgdasens/PT6H/2021-12-30/` and probably `gfs/diag/prgdasens_lgetkf/PT6H/2021-12-30/` (until ens. is handled properly)

This is storing them as if they are input obs to the system.

But I'm fine with not making this change now as it doesn't impact cycling, only archival of the diags.